en/docs/guides/mfa/mfa-for-management-console.md

```diff
@@ -0,0 +1,97 @@
+# Configuring multi-factor authentication for the management console
+
+## Prerequisites
+You need to [create a service provider](../applications/register-sp.md).
+
+## Configure MFA for the management console
+
+To configure MFA for the management console, initially:
+
+1. On the management console, go to **Main** > **Identity** > **Service Providers** > **List**.
+
+2. Click **Edit** on the service provider you wish to configure.
+
+    Configuration of MFA for the management console contains three configuration sections as follows:
+
+    1. [Inbound authentications configurations](#inbound-authentications-configurations)
+    2. [Local and outbound Authentication configurations](#local-and-outbound-authentication-configurations)
+    3. [Local configurations on `deployment.toml`](#local-configurations-on-deploymenttoml)
+
+
+### Inbound authentications configurations
+
+To configure inbound authentication for the management console:
+
+3. Expand the **Inbound Authentication Configuration > SAML2 Web SSO Configuration** section and click **Configure**.
+
+4. Enter the following details under **Manual Configuration**:
+
+    | Field name    | Value |
+    |---------------|-------|
+    | Issuer    | carbonServer  |
+    | Assertion Consumer URLs   | https://localhost:9443/acs    |
+    | Enable Response Signing   | Selected  |
+
+5. Click **Register** to save the configurations.
+
+### Local and outbound Authentication configurations
+
+To configure local and outbound authentication for the management console:
+
+1. Expand the **Local and Outbound Authentication Configuration** section and click **Advanced Configuration**.
+
+2. Click **Add Authentication Step** to add an authentication step.
+
+3. For both authentication steps, select the authenticator from **Local Authenticators** and click **+ Add Authenticator** to add the authenticator.
+
+    !!! info
+        - **Use subject identifier from this step** and **Use attributes from this step** will be selected by default. If you wish to set multiple steps, you can have only one step as the subject step and one as the attribute step.
+        - Note that you cannot use `jwt-basic` option as an authentication step.
+
+4. Click **Update** to save the configurations.
+
+5. Enable the following options:
+    - Use tenant domain in local subject identifier
+    - Use user store domain in local subject identifier
+
+    !!! info
+        Leave the default configurations as it is.
+
+6. Click **Update** to save the configurations.
+
+### Local configurations on `deployment.toml`
```

Comment:
This should be a level 2 heading - `Configure WSO2 IS` or `Configure the server`